docs/contribution_details/CONTRIBUTING_GUIDE.md

```diff
@@ -0,0 +1,28 @@
+# Contributing to LibreLingo
```

Comment:
I think this file could just be docs/CONTRIBUTING_GUIDE.md, no need to put it in a folder